oak_attestation/src/lib.rs

```diff
@@ -20,19 +20,14 @@ extern crate alloc;
 
 pub mod proto {
     pub mod oak {
-        pub mod attestation {
-            pub mod v1 {
-                #![allow(dead_code)]
-                include!(concat!(env!("OUT_DIR"), "/oak.attestation.v1.rs"));
-            }
-        }
-        pub use oak_crypto::proto::oak::crypto;
         pub mod session {
             pub mod v1 {
                 #![allow(dead_code)]
                 include!(concat!(env!("OUT_DIR"), "/oak.session.v1.rs"));
             }
         }
+        pub use oak_crypto::proto::oak::crypto;
+        pub use oak_proto_rust::oak::attestation;
```

Comment:
The reason for the re-exports here is that `session` proto has a dependency on the crypto and attestation. If I won't add it here, `build.rs` would fail.

cc @tiziano88

Reply:
No issues in merging as-is then to avoid blocking other progress, but for the sake of discussion, why does `session` need to be built here and not in `oak_proto_rust`?

Reply:
I think it could, if proto rust generates gRPC code as well

Reply:
Yeah I guess similar question for micro RPC. Are we going to generate client gRPC + server gRPC + client microRPC + server microRPC all in this crate?